web/app.py

```diff
@@ -65,7 +65,10 @@
 )
 
 ap_src_metadata_fetcher = metadata_manager.APSourceMetadataFetcher(
-    ap_repo=repo
+    ap_repo=repo,
+    caching_enabled=True,
```

Comment:
Does this all work with caching disabled still?

Reply:
Yes, I tested that. It works.

Reply:
Does this answer your query @peterbarker?